engines/director/movie.cpp

```diff
@@ -240,7 +240,7 @@ bool Movie::loadArchive() {
 		delete r;
 	}
 
-	_score->setSpriteCasts();
+	// _score->setSpriteCasts(); TODO: Dynamic loading change this
```

Comment:
You need to call this thing after every frame loading now—just the inner loop of this method.